playbooks/roles/crafty-bot/defaults/main.yml

```diff
@@ -0,0 +1,78 @@
+---
+APP_PATH: '/crafty'
+
+USE_SSL: true
+CRAFTY_NGINX_CONFIG_PATH: /etc/nginx/sites-available/crafty
+
+CI_SSH_PUBLIC_KEY: 'your-CI-key'
+
+# Base64 encoded user:password
+docker_registry_token: set-me-please
+
+API_HOSTNAME: "backend.server"
+
+# Settings for Crafty Bot. See crafty_bot/config/settings/ for details.
+ENV_TOKENS:
+  # General
+  # ------------------------------------------------------------------------------
+  # DJANGO_READ_DOT_ENV_FILE: True
+  DJANGO_SETTINGS_MODULE: config.settings.production
+  DJANGO_SECRET_KEY: set-me-please
+  DJANGO_ADMIN_URL: admin/
+  DJANGO_ALLOWED_HOSTS: .crafty.opencraft.com
```

Comment:
Nit: this is already specified in the private vars, so we can make the value more generic here.